mibidata/mibi_image.py

```diff
@@ -235,28 +235,7 @@ def aperture(self):
 
     @aperture.setter
     def aperture(self, value):
-        if value in APERTURE_MAP.values() or value is None:
-            # Allow valid aperture codes or None
-            self._aperture = value
-        else:
-            # Convert known string aperture parameters, if possible
-            try:
-                self._aperture = {
-                    **_DEPRECATED_APERTURE_MAP,
-                    **APERTURE_MAP
-                }[value]
-                warnings.warn(
-                    'Deprecated aperture code \'{}\', converting to \'{}\'. In '
-                    'a future version, values from the following map will be '
-                    'required: {}'.format(value,
-                                          _DEPRECATED_APERTURE_MAP[value],
-                                          APERTURE_MAP)
-                )
-            except KeyError:
-                raise ValueError(
-                    'Invalid aperture code \'{}\', must use values'
-                    'from the following map: {}'.format(value, APERTURE_MAP)
-                )
+        self._aperture = MibiImage.validate_aperture(value)
```

Comment:
Why not `self.validate_aperture`

Reply:
Good point, I also didn't like the name so I changed it.